recipes/neurotic/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "neurotic" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "1dceebe21845828b6783721a82bfd61cd2f87fcca6e373db7ec1555e51ce138e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - neurotic=neurotic.scripts:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - av
+    - ephyviewer >=1.2.0
+    - numpy
+    - packaging
+    - pandas
+    - pyqt
+    - python >=3.6,<3.8  # <3.8 required for pyqtgraph 0.10.0
```

Comment:
I don't see pyqtgraph in the dependencies.

Reply:
pyqtgraph is required only indirectly, through ephyviewer. The pyqtgraph recipe should have the Python<3.8 restriction built in, but it doesn't. I've already opened a PR (conda-forge/pyqtgraph-feedstock#11) to fix that, and if it's accepted the explicit restriction to <3.8 won't be needed here anymore. For now, I need it.

Reply:
Ahhh ok.